libs/models/src/lib/state/player.ts

```diff
@@ -0,0 +1,28 @@
+import { AppRaceId } from './race';
```

Comment:
Add consistent spacing between the type definitions
Eg: Empty line between each new type definition

Reply:
Add consistent spacing 755e6524b5cc463844463d50060be3604003a6a2